go.mod

```diff
@@ -84,6 +84,8 @@ replace (
 	// d1d5f6476656c6aad457e2a91d3436e66b6f2251 is more suitable since it does not break fsouza/go-clientdocker,
 	// yet provides the same fix.
 	github.com/docker/docker => github.com/docker/docker v1.4.2-0.20191121165722-d1d5f6476656
+	// TODO: REMOVE: FAKE BUMP FOR TESTING
+	github.com/openshift/library-go => github.com/sallyom/library-go v0.0.0-20200424135032-af2fa04036d6
```

Comment:
Fix this!

Reply:
yup this is in a TOBEREMOVED commit - I need this to test the change until that library-go PR merges

Reply:
actually i don't need that change so removed the bump : )